datadog_checks_base/requirements.in

```diff
@@ -2,13 +2,15 @@ aws-requests-auth==0.4.2
 binary==1.0.0
 botocore==1.13.42
 contextlib2==0.6.0; python_version < '3.0'
+cryptography==2.8
 ddtrace==0.32.2
 enum34==1.1.6; python_version < '3.0'
 ipaddress==1.0.22; python_version < '3.0'
 kubernetes==8.0.1
 orjson==2.6.1; python_version > '3.0'
 prometheus-client==0.7.1
 protobuf==3.7.0
+pyjwt==1.7.1
```

Comment:
To avoid installing in every check's test suite, let's add to:

- https://github.com/DataDog/integrations-core/blob/eb4a8ff10ebee6928d733321aa85dad548dde8a6/datadog_checks_base/setup.py#L66
- https://github.com/DataDog/integrations-core/blob/eb4a8ff10ebee6928d733321aa85dad548dde8a6/datadog_checks_base/setup.py#L70